File: src/main/java/um/tesoreria/core/hexagonal/facturaPendiente/domain/model/FacturaPendiente.java
package um.tesoreria.core.hexagonal.facturaPendiente.domain.model;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import java.math.BigDecimal;
import java.time.OffsetDateTime;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class FacturaPendiente {
    private Long proveedorMovimientoId;
    private String razonSocial;
    private String cuit;
    private String cbu;
    private OffsetDateTime fechaComprobante;
    private OffsetDateTime fechaVencimiento;
    private String observaciones;
    private String comprobante;
    private Byte debita;
    private Integer prefijo;
    private Long numeroComprobante;
    private BigDecimal importeFactura;
    private BigDecimal importePagado;
}

File: src/main/java/um/tesoreria/core/kotlin/model/view/FacturaPendiente.kt
package um.tesoreria.core.kotlin.model.view

import com.fasterxml.jackson.annotation.JsonFormat
import jakarta.persistence.Entity
import jakarta.persistence.Id
import jakarta.persistence.Table
import org.hibernate.annotations.Immutable
import java.math.BigDecimal
import java.time.OffsetDateTime

@Entity
@Table(name = "vw_factura_pendiente")
@Immutable
data class FacturaPendiente(

    @Id
    val proveedorMovimientoId: Long?,
    val razonSocial: String,
    val cuit: String,
    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ssZ", timezone = "UTC")
    val fechaComprobante: OffsetDateTime?,
    val comprobante: String,
    val prefijo: Int,
    val numeroComprobante: Long,
    val importeFactura: BigDecimal,
    val ordenPagoId: Long?,

    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ssZ", timezone = "UTC")
    val fechaOrdenPago: OffsetDateTime?,
    val prefijoOrdenPago: Int?,
    val numeroOrdenPago: Long?,
    val importeOrdenPago: BigDecimal?,
    val importePagado: BigDecimal?
)